src/CommunityToolkit.Maui.MediaElement/Interfaces/IMediaElement.cs

```diff
@@ -83,6 +83,15 @@ public interface IMediaElement : IView, IAsynchronousMediaElementHandler
 	/// </summary>
 	bool ShouldShowPlaybackControls { get; set; }
 
+	/// <summary>
+	/// Gets or sets the font to use for the subtitle text.
+	/// </summary>
+	string SubtitleFont { get; set; }
+	/// <summary>
+	/// Gets or sets the URL of the subtitle file to display.
+	/// </summary>
+	string SubtitleUrl { get; set; }
```

Comment:
From Maui architecture, the interfaces members just implement the `set` if the platform can set the value. For example, imagine an Entry control, the `Text` property can be set when user types in the device's keyboard, so the platform can set the value and because of that it implements the `set`.

My question is, can the platform sets this property (and others)?

Reply:
Removed set for Subtitle items. It was not needed.